tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Update Description 5322.feature

```diff
@@ -62,7 +62,14 @@ Scenario: 6. A non buyer user cannot update an orders description
         | Another Description |
 
 @5322
-Scenario: 7. Service Failure
+Scenario: 7. A buyer user cannot update an orders description for an organisation they don't belong to
+     Given the user is logged in with the Buyer role for organisation e6ea864e-ef1b-41aa-a4d5-04fc6fce0933
+    When the user makes a request to update the description with the ID C000014-01
+        | Description         |
+        | Another Description |

```

Comment:
Do we need an additional check here to validate the status code of the response, ensuring it is forbidden?